tslint.json

```diff
@@ -0,0 +1,15 @@
+{
+    "extends": ["tslint-config-airbnb", "tslint-config-prettier"],
+    // rules override:
+    "rules": {
+      "no-increment-decrement" : false,
+      "import-name": false, // 23.6,
+      "max-line-length": false,  // 18.12   We can debate about that...
+      "variable-name": [true, "ban-keywords", "check-format", "allow-snake-case", "allow-leading-underscore", "allow-pascal-case"],
+      "ter-indent": [true, 4],
+      "ter-prefer-arrow-callback": false
+    },
+    "linterOptions": {
+      "exclude": ["proto"]
+    }
+  }
```

Comment:
change to eslint

Reply:
Can you send me a cheat sheet or maybe a quick guide?

Reply:
https://code.visualstudio.com/api/advanced-topics/tslint-eslint-migration